packages/astro/test/units/content-collections/get-entry-type.test.js

```diff
@@ -33,52 +33,58 @@ describe('Content Collections - getEntryType', () => {
 			it('Returns "content" for Markdown files', () => {
 				for (const entryPath of ['blog/first-post.md', 'blog/first-post.mdx']) {
 					const entry = fileURLToPath(new URL(entryPath, contentPaths.contentDir));
-					const type = getEntryType(entry, contentPaths, contentFileExts);
+					const type = getEntryType(entry, contentPaths, contentFileExts, false);
```

Comment:
Please add a TODO to this file to remind us to remove this `false` once the assets are no longer experimental. Since TS doesn't type-check this file, we risk forgetting it,